tests/scala/scala3/BUILD

```diff
@@ -40,5 +40,5 @@ scala_library(
 scala_library(
     name = "example-4",
     srcs = ["Example4.scala"],
-    scala = "//scala:3",
+    scala = "//scala:2_13",
```

Comment:
How come you change this?

Reply:
So I have both scala 2 code depending on 3 and scala 3 depending on 2 in this test.